submissions/israelOladosu/htmlForms/ContactForm.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+ <meta charset="UTF-8">
+ <title>Contact Form Page</title>
+</head>
+
+<body>
+ <header>
+  <!-- container for the site logo -->
+  <div>
+   <a href="index.html">
+    <img src="../introductionToHtml/img/Farmville flat green.png" alt="Farmville">
+   </a>
+  </div>
+  <!-- the search and login bar -->
+  <form>
+   <input type="text" placeholder="search">
+   <button>Search</button>
+  </form>
+  <ul>
+   <li><a href="#">SignUp</a></li>
+   <li><a href="#">Contact Us</a></li>
+   <li><a href="#">Cart</a></li>
+  </ul>
+ </header>
+ <hr>
+ <!--Contact Form-->
+ <fieldset>
+  <legend>Contact Form</legend>
+  <h3>You can reach us here!</h3>
+  <form>
+   <label for="fullName">Full Name</label><br>
+   <input type="text" name="fullName" id="fullName" placeholder="Full Name"><br><br>
+   <label for="email">Email</label><br>
+   <input type="email" name="email" id="email" placeholder="Email"><br><br>
+   <label for="subject">Subject</label><br>
+   <input type="text" name="subject" id="subject" placeholder="Subject"><br><br>
+   <label for="massageInput">Massage</label><br>
```

Comment:
There is a typo here `massageInput` instead of `messageInput`. You would need to update this pull request and check the other usage of the typo @Israel-Force